environments/icds-cas/public.yml

```diff
@@ -41,7 +41,7 @@ etc_hosts_lines:
   - '100.71.181.10		cas.commcarehq.org'
   - '100.71.181.10		reports.icds-cas.gov.in'
   - '100.71.181.10		www.icds-cas.gov.in'
-  - '100.126.21.226     mowcdmum.ipstorage.tatacommunications.com'
+  - '100.126.21.242     mowcdmum.ipstorage.tatacommunications.com'
```

Comment:
was this intentional? I don't think it's correct:
```
(icds) skelly@pgshard10:~$ ping mowcdmum.ipstorage.tatacommunications.com
PING mowcdmum.ipstorage.tatacommunications.com (100.126.21.226) 56(84) bytes of data.
```

Reply:
from TCL firewall "mowcdmum.ipstorage.tatacommunications" resolving to 100.126.21.242 and I have changed on elasticsearch servers 

Reference:
https://ipcloud.tatacommunications.com/catalyst/#/request-details/4021083